Docker/Dockerfile

```diff
@@ -1,26 +1,46 @@
 FROM library/python:3.7.0-alpine3.8
 
-RUN apk add build-base
-RUN apk add portaudio-dev
-RUN apk add sqlite
-RUN pip install webob
+LABEL maintainer="istvan@codekuklin.com"
 
-COPY anki-sync-server /app/anki-sync-server
+ARG SERVER_COMMIT='8551021f2b2356e3c3449c1ea6b9745f0856af66'
```

Comment:
You should move this to above the label command and include it in the labels. I would rename it to match the Anki Sync Server environment variables. See https://github.com/ankicommunity/anki-sync-server#envvar-configuration-overrides.

Also remove the defaults.

Reply:
ARG variables will only exist in build time and not after that. Or let's just say they are temporary variables compared to ENV or LABEL. https://docs.docker.com/engine/reference/builder/#arg
But this article explains even better than the original doc: https://vsupalov.com/docker-arg-env-variable-guide/
_"ARG is only available during the build of a Docker image (RUN etc), not after the image is created and containers are started from it... "_

Reply:
So the thought behind this comment was two fold:

1) How can I check the SERVER_COMMIT after the image is built?

You're right ARG variables are only available during the build time. But I thought it would be nice to have this information available in the final image. My suggest was something like this:

```
ARG SERVER_COMMIT='...'
LABEL anki.server.commit="${SERVER_COMMIT}"
```

2) what happens if I don't provide the SERVER_COMMIT?

My thoughts would be that if a commit was not specified, it should use the latest commit. 

There's just ideas and you seem to know your stuff so I'ld love to get your opinions on them